packages/types/src/localization.ts

```diff
@@ -0,0 +1,228 @@
+  badge__thisDevice: LocalizationValue;
+  badge__default: LocalizationValue;
+  badge__unverified: LocalizationValue;
+  badge__requiresAction: LocalizationValue;
+  signUp: {
+    start: {
+      title: LocalizationValue;
+      subtitle: LocalizationValue;
+      actionText: LocalizationValue;
+      actionLink: LocalizationValue;
+    };
+    emailLink: {
+      title: LocalizationValue;
+      subtitle: LocalizationValue;
+      formTitle: LocalizationValue;
+      formSubtitle: LocalizationValue;
+      resendButton: LocalizationValue;
+    };
+    emailCode: {
+      title: LocalizationValue;
+      subtitle: LocalizationValue;
+      formTitle: LocalizationValue;
+      formSubtitle: LocalizationValue;
+      resendButton: LocalizationValue;
+    };
+    phoneCode: {
+      title: LocalizationValue;
+      subtitle: LocalizationValue;
+      formTitle: LocalizationValue;
+      formSubtitle: LocalizationValue;
+      resendButton: LocalizationValue;
+    };
+    continue: {
+      title: LocalizationValue;
+      subtitle: LocalizationValue;
+      actionText: LocalizationValue;
+      actionLink: LocalizationValue;
+    };
+  };
+  signIn: {
+    start: {
+      title: LocalizationValue;
+      subtitle: LocalizationValue;
+      actionText: LocalizationValue;
+      actionLink: LocalizationValue;
+    };
+    password: {
+      title: LocalizationValue;
+      subtitle: LocalizationValue;
+      actionLink: LocalizationValue;
+    };
+    emailCode: {
+      title: LocalizationValue;
+      subtitle: LocalizationValue;
+      formTitle: LocalizationValue;
+      formSubtitle: LocalizationValue;
+      resendButton: LocalizationValue;
+    };
+    emailLink: {
+      title: LocalizationValue;
+      subtitle: LocalizationValue;
+      formTitle: LocalizationValue;
+      formSubtitle: LocalizationValue;
+      resendButton: LocalizationValue;
+    };
+    phoneCode: {
+      title: LocalizationValue;
+      subtitle: LocalizationValue;
+      formTitle: LocalizationValue;
+      formSubtitle: LocalizationValue;
+      resendButton: LocalizationValue;
+    };
+    phoneCode2Fa: {
+      title: LocalizationValue;
+      subtitle: LocalizationValue;
+      formTitle: LocalizationValue;
+      formSubtitle: LocalizationValue;
+      resendButton: LocalizationValue;
+    };
+    alternativeMethods: {
+      title: LocalizationValue;
+      actionLink: LocalizationValue;
+      blockButton__emailLink: LocalizationValue;
+      blockButton__emailCode: LocalizationValue;
+      blockButton__phoneCode: LocalizationValue;
+      blockButton__password: LocalizationValue;
+      getHelp: {
+        title: LocalizationValue;
+        content: `If you’re experiencing difficulty signing into your account, email us and we will work with you to restore access as soon as possible.`;
+        blockButton__emailSupport: LocalizationValue;
+      };
+    };
+  };
+  userProfile: {
+    headerTitle__account: LocalizationValue;
```

Comment:
These keys are used in the UserProfile homepage. I'm thinking of nesting them under `userProfile.home` key so they feel more similar with the other UserProfile pages (eg: `userProfile.usernamePage`).

Thoughts?